bot.py

```diff
@@ -40,6 +41,28 @@
 with open("knowledge/specific.json", "r", encoding='utf-8') as f:
     kattis_specific = json.load(f)
 
+##### ?snipe
+class DeletedMsg:
+    def __init__(self, msg):
+        self.msg = msg
+        self.time = datetime.utcnow()
+
+deleted_messages = {} # {channel id : DeletedMsg}
+
+@client.event
+async def on_message_delete(message):
+    deleted_messages[message.channel.id] = DeletedMsg(message)
+
+@client.command(name='snipe')
```

Comment:
Slash commands are slow to invoke (that's what this is right?) Can we use the legacy thingy